llvm/test/CodeGen/AMDGPU/llvm.amdgcn.workitem.id-unsupported-calling-convention.ll

```diff
@@ -0,0 +1,61 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn -mcpu=gfx942 -O0 -stop-after=si-fix-sgpr-copies -o - %s | FileCheck --check-prefix=SelDAG %s
```

Comment:
Wonder why you want to stop it after `si-fix-sgpr-copies` instead of something earlier? I suppose right after `amdgpu-isel` would be sufficient.

Reply:
works too (see my latest push) :) It just doesn't show the `IMPLICIT_DEF`s explicitly.